app/views/pages/history.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="page-container">
+  <%= stylesheet_link_tag "home" %>
+  <h1>History</h1>
+  <br>
+  <h4>Issues</h4>
+  <br>
+  <% if current_user %>  
+    <div>You have open <%= current_user.issues.length %> issues.</div>
+    <% current_user.issues.each do |issue| %>
```

Comment:
Make sure there is a message if a user hasn't reported any issues yet. We don't want folks getting confused by a blank page.